### ISSUE TITLE
Add support for micronaut 4

### ISSUE THREAD

**Issue by bjor-joh** [slackapi/java-slack-sdk#1252]
There is a [todo](https://github.com/slackapi/java-slack-sdk/blob/main/bolt-micronaut/pom.xml#L13) to upgrade to micronaut 4.x. 
Any concrete plans for this? 

I tried using the existing version with micronaut 4 a while back but it didn't work. 

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ x ] **bolt-micronaut** (Bolt for Java - optional modules)
* [ ] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

**Comment by seratch** [comment]
Hi @bjor-joh, thanks for writing in. We will update the module to support Micronaut 5 in the future, but I cannot tell when the change will be released. The module is quite simple and it has only two source files. If you're in a hurry to use Micronaut 4, please fork the implementation and adjust it to work with v4.

**Comment by hrothwell** [comment]
@bjor-joh I am definitely late to the party here but just came across this issue now. You should be able to get things working using Micronaut's `@Import` (which they say only works in java but seems fine in kotlin for me, possibly they mean the whole jvm or documentation is just out of date)

example:
```kotlin
import io.micronaut.context.annotation.Import
import io.micronaut.runtime.Micronaut

// checks these packages for JSR-330 annotated classes to make beans
// can also just be specific classes, which i did in my own use case
@Import(packages = ["com.slack.api.bolt.micronaut"])
object Application {
  @JvmStatic
  fun main(args: Array<String>) {
    Micronaut.build()
      .packages("com.github.example")
      // ... other setup you might do 
      .mainClass(Application.javaClass)
      .start()
  }
}
```

**Comment by seratch** [comment]
Resolved by #1294 (thanks to @hrothwell!)
